src/index.html

```diff
@@ -7,6 +7,377 @@
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__prices">
+          <p class="card__price">Price:</p>
+
+          <p class="card__value">$2,199</p>
+        </div>
+
+        <a href="#" class="card__link">
+          Buy
+        </a>
+      </div>
+
+      <div class="card catalog__card">
+        <img
+          src="./images/imac.jpeg"
+          alt="Apple iMac 27 A1419"
+          class="card__image"
+        >
+
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="card__subtitle">Product code: 195434</p>
+
+        <div class="card__raiting">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__prices">
+          <p class="card__price">Price:</p>
+
+          <p class="card__value">$2,199</p>
+        </div>
+
+        <a href="#" class="card__link">
+          Buy
+        </a>
+      </div>
+
+      <div class="card catalog__card">
+        <img
+          src="./images/imac.jpeg"
+          alt="Apple iMac 27 A1419"
+          class="card__image"
+        >
+
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="card__subtitle">Product code: 195434</p>
+
+        <div class="card__raiting">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__prices">
+          <p class="card__price">Price:</p>
+
+          <p class="card__value">$2,199</p>
+        </div>
+
+        <a href="#" class="card__link">
+          Buy
+        </a>
+      </div>
+
+      <div class="card catalog__card">
+        <img
+          src="./images/imac.jpeg"
+          alt="Apple iMac 27 A1419"
+          class="card__image"
+        >
+
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="card__subtitle">Product code: 195434</p>
+
+        <div class="card__raiting">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
```

Comment:
```suggestion
            <div class="stars__star"></div>
            <div class="stars__star"></div>
```

Reply:
Hey.
Sorry, but i can't understand what's wrong here?
Only one line between divs.

Reply:
Good morning. I understood you and I had read Task about "adding empty lines between multiline sibling".
In task was told "don't add empty lines between parent and child elements". 
My <div class="card__raiting"> is parent of  my <div class="stars stars--4"> and <div class="stars stars--4"> is  parent of my  <div class="stars__star">.
In this case i don't understand where i must to add empty lines. 
I'll visit Q&A today for beter understanding this case and after will change my code.
Thanks for your comments.
Have a nice day.

Reply:
Well done, your resolved this case right!